src/codelabs/async-await.md

```diff
@@ -144,13 +179,20 @@ This is because `getUserOrder()` delays before it prints "Large Latte".
 Run the following example to see how a future completes with an error.
 A bit later you'll learn how to handle the error.
 
-<iframe
-  src="{{site.dartpad-embed}}?id=d843061bbd9388b837c57613dc6d5125"
-  style="border: 1px solid lightgrey; margin-top: 10px; margin-bottom: 25px"
-  frameborder="no"
-  height="275"
-  width="100%" >
-</iframe>
+<?code-excerpt "async_await/bin/futures_intro.dart (error)" replace="/Error//g"?>
+```dart:run-dartpad:height-300px
+Future<void> getUserOrder() =>
+    // Imagine that this function fetches user info from a data source
+    Future.delayed(
+      Duration(seconds: 2),
```

Comment:
@legalcodes are you OK with the durations changing?

Reply:
To fill you in: I brought it down to 2 secs where the wait was unnecessary, but kept it 4 sec for the example that relies on the 4 sec counter.